extensions/devservices/common/src/main/java/io/quarkus/devservices/common/ConfigureUtil.java

```diff
@@ -3,17 +3,17 @@
 import java.io.IOException;
 import java.io.InputStream;
 import java.io.UncheckedIOException;
-import java.util.ArrayList;
-import java.util.List;
-import java.util.Locale;
-import java.util.Map;
-import java.util.Properties;
+import java.lang.reflect.Field;
+import java.util.*;
```

Comment:
Please avoid star imports (the IDE adds them automatically most likely)

Reply:
Damn i was pretty sure I ran mvn -Dqwickly...
Will correct this.